source/agora/common/Task.d

```diff
@@ -51,4 +58,30 @@ public class TaskManager
         static import vibe.core.core;
         vibe.core.core.sleep(dur);
     }
+
+    /***************************************************************************
+
+        Run an asynchronous task after a given time.
+
+        The task will first run after the given `timeout`, and
+        can either repeat or run only once (the default).
+        Use the `setTimer` vibe.d's
+
+        Params:
+            timeout = Determines the minimum amount of time that elapses before
+                the timer fires.
+            dg = This delegate will be called when the timer fires.
+            periodic = Speficies if the timer fires repeatedly or only once.
```

Comment:
```suggestion
            periodic = Specifies if the timer fires repeatedly or only once.
```

Reply:
https://github.com/vibe-d/vibe-core/blob/872bff8472cc6755e4d0424b347a19a4f5d86ec3/source/vibe/core/core.d#L876

Copy and paste problems :(